ocr_service/api/endpoints/extraction.py

```
@@ -0,0 +1,294 @@
+from fastapi import APIRouter, HTTPException, File, UploadFile
+from pydantic import BaseModel
+import tempfile
+import os
+import logging
+import requests
+from typing import Dict, Any
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)